specs/errors-spec.js

```diff
@@ -0,0 +1,127 @@
+const { switchToQASpace } = require('../utils')
+
+describe('The Example App - Error pages', () => {
+  context('non QA', () => {
+    it('Detects invalid credentials and shows settings form', () => {
+      cy.request({
+        url: '/?space_id=1&delivery_token=2&preview_token=3'
+      }).then((response) => {
+        expect(response.status).to.equal(200)
```

Comment:
Should this be a 200?

Reply:
yes it should be, since it is more a form validation error, not a real server error